Widget/app/src/androidTest/java/com/microsoft/device/display/samples/widget/XmlParserRssFeedTest.kt

```diff
@@ -0,0 +1,243 @@
+package com.microsoft.device.display.samples.widget
+
+import androidx.test.filters.MediumTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.microsoft.device.display.samples.widget.feed.RssFeed
+import com.microsoft.device.display.samples.widget.feed.RssItem
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.xmlpull.v1.XmlPullParserFactory
+import java.io.StringReader
+import org.hamcrest.Matchers.`is` as iz
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@MediumTest
+class XmlParserRssFeedTest {
```

Comment:
This test class should be probably a unit test class since we are testing the parser and not UI or dealing with the framework I believe.

Reply:
The XmlParserRssFeedTest class contains instrumented unit tests because they are using XmlPullParserFactory and XmlPullParser which are included in the Android framework and are needed in parseXml(), handleStartTagElement(), parseRequiredElement() methods.

Reply:
Good catch! totally missed that. Thanks Bianca :-)